### PR TITLE
USHIFT-499: Replace k8s pause image with openshift pod image

### PR DESCRIPTION
**Pull request by zshi-redhat**
Related to [USHIFT-499](https://issues.redhat.com//browse/USHIFT-499)

### CODE REVIEW

File: pkg/release/release.go [hunk 1]
```diff
@@ -30,6 +30,6 @@ var Image = map[string]string{
 	"ose_csi_node_registrar":    "quay.io/microshift/ose-csi-node-driver-registrar" + Base,
 	"ose_csi_livenessprobe":     "quay.io/microshift/ose-csi-livenessprobe" + Base,
 	"ovn_kubernetes_microshift": "quay.io/microshift/ovn-kubernetes-microshift:" + Base,
-	"pause":                     "quay.io/microshift/pause:" + Base,
+	"pod":                       "quay.io/microshift/pause:" + Base,
```
Comment (ggiguash) [inline pkg/release/release.go:33]: If "pause" is not unique enough, can we make it into "pause_pod", not just "pod"?

File: packaging/crio.conf.d/microshift_amd64.conf [hunk 2]
```diff
@@ -15,3 +15,4 @@ plugin_dirs = [
 # for community builds on top of OKD, this setting has no effect
 [crio.image]
 global_auth_file="/etc/crio/openshift-pull-secret"
+pause_image = "registry.k8s.io/pause:3.6"
```
Comment (ggiguash) [inline packaging/crio.conf.d/microshift_amd64.conf:18]: Shouldn't we add the OCP image here already?

File: packaging/crio.conf.d/microshift_arm64.conf [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+plugin_dirs = [
+        "/usr/libexec/cni",
+        "/opt/cni/bin"
+]
+
+# the pull secret is mandatory for MicroShift builds on top of OpenShift
+# for community builds on top of OKD, this setting has no effect
+[crio.image]
+global_auth_file="/etc/crio/openshift-pull-secret"
+pause_image = "registry.k8s.io/pause:3.6"
```
Comment (ggiguash) [inline packaging/crio.conf.d/microshift_arm64.conf:18]: Shouldn't we add the OCP image here already?

**Review by ggiguash — CHANGES_REQUESTED** [review]

**Comment by ggiguash** [comment]
/retitle USHIFT-499: Replace k8s pause image with openshift pod image

File: pkg/release/release.go (hunk 1 above)
Comment (zshi-redhat) [reply pkg/release/release.go:33]: @ggiguash 'pod' is the exact image name in release payload, shall we keep as it is?

File: packaging/crio.conf.d/microshift_amd64.conf (hunk 2 above)
Comment (zshi-redhat) [reply packaging/crio.conf.d/microshift_amd64.conf]: fixed.

File: packaging/crio.conf.d/microshift_arm64.conf (hunk 3 above)
Comment (zshi-redhat) [reply packaging/crio.conf.d/microshift_arm64.conf]: fixed.

File: pkg/release/release.go (hunk 1 above)
Comment (ggiguash) [reply pkg/release/release.go:33]: I'm not sure. It's too generic for me, hence the comment. 
Anyway, let's see what other reviewers think.

File: scripts/auto-rebase/rebase.sh [hunk 4]
```diff
@@ -378,6 +378,11 @@ update_images() {
                 awk "!/\"${i}\"/ {print \$0} /\"${i}\"/ {printf(\"\\t\\t%-${w}s  %s\n\", \"\\\"${i}\\\":\", \"\\\"${digest}\\\",\")}" \
                     "${REPOROOT}/pkg/release/release_${arch}.go" > t
                 mv t "${REPOROOT}/pkg/release/release_${arch}.go"
+                if [[ "$i" == "pod" ]]; then
+                    echo "Updating image pod (${arch}) in packaging/crio.conf.d/microshift_${arch}.conf"
```
Comment (ggiguash) [inline scripts/auto-rebase/rebase.sh:382]:
```suggestion
                    echo "Updating pause_image pod (${arch}) in packaging/crio.conf.d/microshift_${arch}.conf"
```

**Comment by ggiguash** [comment]
This is LGTM for me. 
Before merging, I'd like to see other reviewers' comments about the pause pod name in release.go files.

**Review by ggiguash — CHANGES_REQUESTED** [review]

File: packaging/crio.conf.d/microshift_amd64.conf [hunk 5]
```diff
@@ -15,3 +15,4 @@ plugin_dirs = [
 # for community builds on top of OKD, this setting has no effect
 [crio.image]
 global_auth_file="/etc/crio/openshift-pull-secret"
+pause_image = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:c66675912585b7da1f5e33d4249cb2424e6d155a9038bbd393f3e4456329b827"
```
Comment (dhellmann) [inline packaging/crio.conf.d/microshift_amd64.conf:18]: Won't we need to be able to update this as part of the rebase process?

**Comment by dhellmann** [comment]
This is good. I was thinking of templating out the config file when we build the RPM, but doing it this way makes it easier to run from source, so that's good.

/lgtm

**Pull request merged**